### PR TITLE
Added a cron sample

### PR DESCRIPTION
**Pull request by meteatamel**

### CODE REVIEW

**Comment by meteatamel** [comment]
Anyone knows what the failure is about? I couldn't tell what's wrong from the logs.

**Comment by tswast** [comment]
@meteatamel The Travis failure is because Travis only allows decrypting service keys for pull requests from branches, not forks. I'll run the tests manually to see if they pass.

**Comment by tswast** [comment]
Tests pass for me, locally. LGTM.

File: managed_vms/cron/pom.xml [hunk 1]
```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (lesv) [inline managed_vms/cron/pom.xml]: License & Copyright

File: managed_vms/cron/src/main/appengine/app.yaml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+runtime: java
```
Comment (lesv) [inline managed_vms/cron/src/main/appengine/app.yaml]: License & copyright

File: managed_vms/cron/src/main/appengine/app.yaml [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+vm: true
```
Comment (lesv) [inline managed_vms/cron/src/main/appengine/app.yaml]: License & copyright

File: managed_vms/cron/src/main/appengine/app.yaml [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+
```
Comment (lesv) [inline managed_vms/cron/src/main/appengine/app.yaml]: License & copyright

File: managed_vms/cron/src/main/appengine/cron.yaml [hunk 5]
```diff
@@ -0,0 +1,4 @@
+cron:
```
Comment (lesv) [inline managed_vms/cron/src/main/appengine/cron.yaml]: License & copyright

**Pull request merged**